manager/controlapi/ca_rotation.go

```diff
@@ -0,0 +1,216 @@
+package controlapi
+
+import (
+	"bytes"
+	"context"
+	"time"
+
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+
+	"github.com/cloudflare/cfssl/helpers"
+	"github.com/docker/swarmkit/api"
+	"github.com/docker/swarmkit/ca"
+	"github.com/docker/swarmkit/log"
+)
+
+var minRootExpiration = helpers.OneYear
```

Comment:
My first instinct was: this is reasonable. But on further consideration, and given that fact that:
- This is the root CA
- We have the ability of rotating at any time if compromised

This makes me think that maybe we should help our users not shoot themselves in the foot and increase the minimum a bit more (3 years)? Just a thought though, I'm 100% ok with 1 year.

/cc @sul3n3t